examples/compass.py

```diff
@@ -0,0 +1,21 @@
+from microbit import *
+
+needles = (Image.CLOCK12, Image.CLOCK1, Image.CLOCK2, Image.CLOCK3,
+           Image.CLOCK4, Image.CLOCK5, Image.CLOCK6, Image.CLOCK7, 
+           Image.CLOCK8, Image.CLOCK9, Image.CLOCK10, Image.CLOCK11)
+
+#You need to spin the microbit about a few times to help calibrate the compass
```

Comment:
@markshannon Cool example. Perhaps for user friendliness in the example, display a message to indicate "Calibrating..." before the while loop output.